CONTRIBUTING.md

```diff
@@ -251,46 +251,94 @@ Each library needs to provide a `ApiCompatVersion` property which is set to the
 ### Releasing a new version of a GA'ed libary
 Since the [eng/Packages.Data.props](https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Packages.Data.props) is currently maintained manually, you will need to update the version number for your library in this file when releasing a new version.
 
-## Dev Feed
-We publish daily built packages to a [dev feed](https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-net) which can be consumed by adding the dev feed as a package source in Visual Studio. 
+## NuGet Package Dev Feed
 
-Follow instructions provided [here](https://docs.microsoft.com/nuget/consume-packages/install-use-packages-visual-studio#package-sources) and use `https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json` as the source.
+The Azure SDK for .NET releases packages daily from our CI pipeline to our NuGet package dev feed to help developers use and test new libraries before they are officially released to NuGet.
 
-You can also achieve this from the command line.
+**Dev Feed Package Browser**:
 
-```nuget.exe sources add -Name "Azure SDK for Net Dev Feed" -Source "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"```
+1. https://aka.ms/azsdk/net/feed
```

Comment:
Do you really want to include both links given you made aka links?

Reply:
I was thinking that some users would want short and others would want to see what the URL is.

Reply:
Given that most people will simply copy and paste this link I wouldn't even suggest we hide it behind an aka link. Having the full link allows folks to better identify where it is coming from.